boards/pba-d-01-kw2x/doc.txt

```diff
@@ -9,29 +9,54 @@ Designed and produced by PHYTEC Messtechnik GmbH, D-55129 Mainz.
-and the
-[phyNODE](https://github.com/PHYTEC-Messtechnik-GmbH/phynode-riot-examples/wiki/phyNODE-Characteristics)
- * [Toolchain, build and debug Information](https://github.com/PHYTEC-Messtechnik-GmbH/phynode-riot-examples/wiki/RIOT-getting-started#toolchain-summary)
- * [A detailed step-by-step Guide that explains how to set up the Environment](https://github.com/PHYTEC-Messtechnik-GmbH/phynode-riot-examples/wiki/RIOT-getting-started#step-by-step-installation-guide)
+
+The phyNODE evaluation board consists of a [phyWAVE](https://www.phytec.de/produkte/internet-of-things/phywave/)
+module mounted on a common PCB with several sensor and actuator modules.
+The board provides 2 USB ports, one for development (debug and flash) and the
+other to be used by the firmware.
+
 The implementation status of the phyWAVE and phyNODE peripherals are listed
 below. You can find links to the corresponding Pull Requests which may give you
 a code reference.
 
 <img src="https://raw.githubusercontent.com/jremmert-phytec-iot/Pictures_Phytec_IOT/master/EvalBoard_3.png" width="800" />
 phyNODE board with equipped phyWAVE-KW2x processor/radio-module.
 
-### Details
-The PCB-pieces with the mounted sensors can be separated from the evaluation
-board to capture sensor values from specific spatial points.
+@note The PCB-pieces with the mounted sensors can be separated from the
+      evaluation board to capture sensor values from specific spatial points.
+
+## Pinout Reference
+
+The board features an Arduino like header connector to mount shields.The spacing
+of the 4 pin rows is a bit offset, thus shields might not fit perfectly.
+
+@note Not all signals and pins are compatible with Arduino Uno R3. Also check
+      the pin configuration in RIOT, i.e., see `periph_conf.h` of the board.
 
-[Pinout reference](https://github.com/PHYTEC-Messtechnik-GmbH/phynode-riot-examples/wiki/Interfaces)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+                             _X1____________________
+                            |                   D15 |___DIO21, PE19, I2C0 SCL
+                            |                   D14 |___DIO20, PE18, I2C0 SDA
+                            |                  AREF |___AREF,  VCC3V3
+                            |                  GND3 |___GND
+                        X___| RESERVED          D13 |___DIO3,  PC5,  SPI0 SCK
+                   VCC3V3___| IOREF             D12 |___DIO5,  PC7,  SPI0 MISO
+                   nRESET___| RST TGTMCU        D11 |___DIO4,  PC6,  SPI0 MOSI
+                   VCC3V3___| P3V3              D10 |___DIO2,  PC4,  SPI0 CS0
+              P5V USB OUT___| P5V USB            D9 |___DIO9,  PD4,  PWM
+                GND_________| GND2               D8 |___DIO6,  PD1
+                        \___| GND1                  |
+                        X___| P5-9V VIN          D7 |___DIO24, PA1,  PWM,  TDI2
```

Comment:
Just looked at the generated doc HTML and it looks like the P5 is somehow linked to an MSP430 header file. No idea why and how to remove that, but that should be avoided to not confuse people.

Reply:
Doxygen says a preceeding `%` should avoid auto link, however that doesn't work in code block (at least not for me), so I changed it from  dash `-` to underscore `_` to trick it.

Reply:
yep looks better now